lib/common/logging.c

```diff
@@ -710,6 +717,19 @@ crm_priority2int(const char *name)
 }
 
 
+/*!
+ * \brief Set PCMK_service environment variable
```

Comment:
also \internal

Good guess for the summary, but that's actually the least important thing it does :)

Setting crm_system_name is the primary purpose of the function. This is an identifier for the current process (usually the executable name), with the most important uses being:
* Log messages (most importantly it's in the format string as the name listed after the hostname at the start of all messages)
* Name used when connecting as a client to server IPC
* Name used to determine whether certain environment variables are in effect (PCMK_debug, PCMK_blackbox, PCMK_valgrind_enabled, PCMK_callgrind_enabled)

I'm thinking something along the lines of "Set identifier for current process to use in logs, IPC, etc."